### PR TITLE
live: Use current directory when /tmp is failed

### PR DESCRIPTION
**Pull request by gonapps**
In some systems like a non-rooted android environment (termux)
You don't have a write access to /tmp directory

Signed-off-by: Byeonggon Lee <gonny952@gmail.com>

### CODE REVIEW

File: cmds/live.c [hunk 1]
```diff
@@ -104,15 +104,23 @@ int command_live(int argc, char *argv[], struct opts *opts)
...
+
+		fd = mkstemp(template + sizeof("/tmp/") - 1);
+
+		if (fd < 0)
+			pr_err("cannot create temp name");
+		tmp_dirname = template + sizeof("/tmp/") - 1;
+	}
 
 	close(fd);
 	unlink(template);
```
Comment (namhyung) [inline cmds/live.c]: You could either overwrite template or use tmp_dirname here (and below).

Comment (gonapps) [reply cmds/live.c]: Modified to tmp_dirname += sizeof("/tmp/") - 1;

**Comment by gonapps** [comment]
@namhyung Updated the commit.

File: cmds/live.c (hunk 1 above)
Comment (gonapps) [reply cmds/live.c]: Thanks. I've fixed now

**Pull request merged**